web_poet/mixins.py

```diff
@@ -14,6 +14,14 @@ def css(self, query) -> parsel.SelectorList:
         """A shortcut to ``.selector.css()``."""
         return self.selector.css(query)  # type: ignore[attr-defined]
 
+    def jmespath(self, query: str, **kwargs) -> parsel.SelectorList:
+        """A shortcut to ``.selector.jmespath()``."""
+        if not hasattr(self.selector, "jmespath"):  # type: ignore[attr-defined]
+            raise AttributeError(
+                "Please install parsel >= 1.8.1 to get jmespath support"
+            )
+        return self.selector.jmespath(query, **kwargs)  # type: ignore[attr-defined]
```

Comment:
@Gallaecio 

> I wonder if we should use a try-except here instead, even if it won’t make much of a difference.

It seems it'd be a bit more code, with the same result.
 ```suggestion
        selector = self.selector
        try:
            jmespath = selector.jmespath
        except AttributeError:
            raise AttributeErrror("please install...")
        return jmespath(query, **kwargs)
```